justfile

```diff
@@ -68,7 +68,7 @@ install-targets:
 # Generates the docs
 generate-docs:
     cargo clean --doc
-    cargo doc --workspace --features "{{all_features}}"
+    cargo doc --workspace --features "{{all_features}}" --no-deps
```

Comment:
I had to disable the deps, because they also contain broken links, which are detected by the newer `cargo-deadlinks` version.